src/notebook/Notebook.jl

```diff
@@ -211,10 +229,21 @@ function load_notebook_nobackup(io, path)::Notebook
             code_raw = String(readuntil(io, _cell_id_delimiter))
             # change Windows line endings to Linux
             code_normalised = replace(code_raw, "\r\n" => "\n")
+            
+            # get the information if a cell is disabled
+            running_disabled = startswith(code_normalised, _running_disabled_prefix)
+            depends_on_disabled_cells = startswith(code_normalised, _depends_on_disabled_cells_prefix)
+
+            # remove the disabled on startup comments for further processing in Julia
+            code_normalised = replace(replace(code_normalised, _running_disabled_prefix => ""), _running_disabled_suffix => "")
+            code_normalised = replace(replace(code_normalised, _depends_on_disabled_cells_prefix => ""), _depends_on_disabled_cells_suffix => "")
+
             # remove the cell suffix
             code = code_normalised[1:prevind(code_normalised, end, length(_cell_suffix))]
 
             read_cell = Cell(cell_id, code)
+            read_cell.running_disabled = running_disabled
+            read_cell.depends_on_disabled_cells = depends_on_disabled_cells || running_disabled
```

Comment:
We don't need to _read_ this behaviour, right? We only need to write it.

Reply:
When I remove the line
```
read_cell.depends_on_disabled_cells = depends_on_disabled_cells || running_disabled
```
I get the error message in the console:
```
┌ Warning: Old Pluto notebook might not have loaded correctly. Backup saved to: 
│   backup_path = "C:\\Users\\benlu\\.julia\\pluto_notebooks\\cell_deactivation_demo backup 2.jl"
└ @ Pluto c:\Users\benlu\.julia\dev\Pluto\src\notebook\Notebook.jl:339
```

Reply:
👍